pkgs/tools/package-management/nix/common.nix

```diff
@@ -87,6 +87,13 @@ in
 , pkgsi686Linux
 , runCommand
 }: let
+
+  # perl (dependency of git/man) doesn't build on pkgsStatic, but since it's just used for tests, we can ignore it
+installCheckInputs = lib.optional atLeast221
```

Comment:
Wouldn't `nativeInstallCheckInputs` also pick the correct versions?

Reply:
I thought it would be different but it's the same in this case.

Reply:
I think the comment is somewhat misleading now that the explicit workaround is gone.